_maps/map_files/burgerstation/burgerstation.dmm

```diff
@@ -2684,8 +2685,8 @@
 /area/station/science)
 "biL" = (
 /obj/machinery/air_sensor/zauker_tank,
```

Comment:
change this to a plasma tank sensor too?

Honestly should probably just be co2 like standard on every other execution room but at least its not zauker